### PR TITLE
Finished AcquireConfig

### PR DESCRIPTION
**Pull request by alexander871015**
Implemented Acquiring config portion of the fetcher

### CODE REVIEW

File: hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc [hunk 1]
```diff
@@ -22,4 +22,18 @@ void HologramDataSourceAvailabilityFetcher::Process() {
     // TODO(alexanderlin): add implementation.
 }
 
+void HologramDataSourceAvailabilityFetcher::
+    AcquireConfig(const std::string& path) {
+    std::ifstream ins(path);
+    std::string config_content;
```
Comment (sarahchen-google) [inline hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc:28]: config_content is not used?

File: hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc [hunk 2]
```diff
@@ -22,4 +22,18 @@ void HologramDataSourceAvailabilityFetcher::Process() {
...
+    google::protobuf::io::IstreamInputStream config_stream(&ins);
+    
+    // Path must be valid.
+    assert(ins.good());
```
Comment (sarahchen-google) [inline hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc:32]: Move it after line 27. Things related should be close with each other.

File: hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.h [hunk 3]
```diff
@@ -43,6 +46,8 @@ class HologramDataSourceAvailabilityFetcher: public HologramDataFetcher {
     HologramConfigSet hologram_configs_;
     std::unordered_map<std::string, HologramDataAvailability> 
         data_sources_availability_map_;
+
+    void AcquireConfig(const std::string& path);
```
Comment (sarahchen-google) [inline hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.h:50]: Move private function above private variables.

File: hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc (hunk 1 above)
Comment (alexander871015) [reply hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc:28]: deleted

File: hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.h [hunk 4]
```diff
@@ -39,6 +42,8 @@ class HologramDataSourceAvailabilityFetcher: public HologramDataFetcher {
     void Process() override;
 
 private:
+    void AcquireConfig(const std::string& path);
```
Comment (emilyqu) [inline hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.h:45]: nit: config_file_path

File: hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc [hunk 5]
```diff
@@ -22,4 +22,16 @@ void HologramDataSourceAvailabilityFetcher::Process() {
     // TODO(alexanderlin): add implementation.
 }
 
+void HologramDataSourceAvailabilityFetcher::
+    AcquireConfig(const std::string& file_config_path) {
+    std::ifstream ins(path);
```
Comment (emilyqu) [inline hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc:27]: should this be config_file_path?

File: hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc [hunk 6]
```diff
@@ -22,4 +22,16 @@ void HologramDataSourceAvailabilityFetcher::Process() {
     // TODO(alexanderlin): add implementation.
 }
 
+void HologramDataSourceAvailabilityFetcher::
+    AcquireConfig(const std::string& file_config_path) {
```
Comment (emilyqu) [inline hologram_dashboard/fetcher/hologram_data_source_availabiliity_fetcher.cc:26]: nit: config_file_path to be consistent with the header.

**Pull request merged**